weave/integrations/crewai/cassettes/crewai_test/test_simple_crew.yaml

```diff
@@ -0,0 +1,5628 @@
+interactions:
+- request:
```

Comment:
This entire first block targeting `https://raw.githubusercontent.com/BerriAI/litellm/main/model_prices_and_context_window.json` can be removed, dramatically reducing this file size